lib/src/channels/http.dart

```diff
@@ -135,9 +142,25 @@ class LoggingHttpClient implements HttpClient {
 
   @override
   Future<HttpClientRequest> getUrl(Uri url) {
-    // todo (pq): consider same logic as open (but w/ GET added)
-    _todo('getUrl');
-    return proxy.getUrl(url);
+    final int id = _nextRequestId++;
+    final String method = 'GET';
+
+    // #1 • GET • https://flutter.io url
+    _log.log('#$id • $method • $url url');
+
+    Future<HttpClientRequest> request = proxy.getUrl(url);
+    return request.then((HttpClientRequest req) {
+      _log.log('#$id • $method • $url request ready');
```

Comment:
I'd be nice to determine which headers we're sending to the server (as well as the ones we get back, below).

Reply:
Agreed.  TODO added.

Reply:
Actually, easy enough to just add, so done.